activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
We could but there's nothing under this line so this is not necessary. And it's still documented, it's just under the `ActiveSupport` namespace (I don't know if you are asking for that or asking whether it should still be present at the root level of the sidebar).